js/index.js

```diff
@@ -176,30 +126,26 @@ messageForm.addEventListener("submit", function(e) {
 
 // get my github repos
 
-const gitHubRequest = new XMLHttpRequest();
 const projectSections = document.querySelector('#projects');
 
-gitHubRequest.open('GET', 'https://api.github.com/users/curiousBellyButton/repos');
-gitHubRequest.send();
 
-gitHubRequest.addEventListener('load', function (){
-  const repositories = JSON.parse(gitHubRequest.responseText)
-  console.log(repositories);
+const gitHubProjectUrl = 'https://api.github.com/users/curiousBellyButton/repos'
 
+fetch(gitHubProjectUrl)
+  .then((response) => {
+    if(!response.ok) throw alert(`status: ` + response.status);
```

Comment:
```suggestion
    if(!response.ok) throw Error(`status: ${response.status}`);
```
... and then, if you want to use `alert`, that can be in an [catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch).